test/E2ETests/E2ETests/Helpers/HttpHelpers.cs

```diff
@@ -19,7 +19,7 @@ public static async Task<HttpResponseMessage> InvokeHttpTrigger(string functionN
             return await GetResponseMessage(request);
         }
 
-        public static async Task<HttpResponseMessage> InvokeHttpTriggerWithBody(string functionName, string body, HttpStatusCode expectedStatusCode, string mediaType, int expectedCode = 0)
```

Comment:
Is there a reason we don't have to have expectedStatusCode checked anymore?

Reply:
`expectedStatusCode` was never used in the method so I just removed it